homeworks/09-node-chokidar/src/Importer.ts

```diff
@@ -0,0 +1,42 @@
+import csvjson from "csvjson"
+import EventEmitter from "events"
+import fs from "fs"
+import fsPromises from "fs/promises"
+import prettyjson from "prettyjson"
+import { DirWatcherEvent } from "./DirWatcher"
+
+export default class Importer {
+
+  constructor(public eventEmitter: EventEmitter) {
+  }
+
+  listen() {
+    this.eventEmitter.on(DirWatcherEvent.Added, (paths: string[]) => this.printFiles("[EVENT] FILE ADDED", paths))
```

Comment:
`carry` can be used here to simplify the first arguments as well. u can try to play with it.